metrics/common.go

```diff
@@ -40,6 +40,18 @@ func NewHistogram(name, subsystem, help string, labels []string) Histogram {
 	return prmkit.NewHistogramFrom(prometheus.HistogramOpts{Namespace: Namespace, Subsystem: subsystem, Name: name, Help: help}, labels)
 }
 
+// NewIntegerHistogram creates a Histogram metrics for integer numbers.
+func NewIntegerHistogram(name, subsystem, help string, labels []string) Histogram {
+	buckets := []float64{0, 1, 2, 5, 10, 20, 50, 100, 500}
```

Comment:
i am not sure that generic integer histogram makes sense. buckets should reflect actual values, sizes of the blocks and tx will have very different values, right? i suggest to define buckets per each metric (or metrics type, e.g extensions) and use prmkit.LinearBucket and prmkit.ExponentialBucket (whichever is more suitable).

Reply:
Something like [this](https://github.com/spacemeshos/go-spacemesh/pull/2806/commits/ff06fb007410b521c8de6354f42d3bf717ae86df)?